src/components/Settings/SettingTile.tsx

```diff
@@ -11,7 +11,7 @@ import { Icon } from "components/Icon"
  * interface representing a setting's UI fields
  */
 export interface SettingOptions {
-  title: string
```

Comment:
sai non mi piace molto l'idea di rendere non obbligatorio il titolo, dovrebbe essere assolutamente obbligatorio, piuttosto dovremmo non passare affatto l'opzione.
direi che in questo senso avere tutte le opzioni in un array da mappare ai componenti non ha molto senso dato che eccetto questo gli altri non sono procedurali, personalmente già ce ci siamo modificherei questa cosa scrivendo a mano a mano tutti questi `SettingTile` nel corpo di `Settings.tsx` invece di generarli da questo `settingsList` che dovrebbe scomparire